CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc

```diff
@@ -253,21 +253,21 @@ void CaloTPGTranscoderULUT::setup(HcalLutMetadata const& lutMetadata, HcalTrigTo
     outputLUT_.resize(theTopology->getHTSize());
     hcaluncomp_.resize(theTopology->getHTSize());
 
+    plan1_towers_.clear();
+    for (const auto& id: lutMetadata.getAllChannels()) {
+       if (not (id.det() == DetId::Hcal and theTopology->valid(id)))
+          continue;
+       HcalDetId cell(id);
+       if (not theTopology->dddConstants()->isPlan1(cell))
+          continue;
+       for (const auto& tower: theTrigTowerGeometry.towerIds(cell))
+          plan1_towers_.emplace(tower);
+    }
```

Comment:
hello @matz-e 
plan1_towers_ 
is filled only at this line, inside CaloTPGTranscoderULUT::getOutputLUTSize(...),
and loadHCALCompress() does not seem to affect its content.

Therefore,
why does the swapping the ordering matter ?